test/face-register.test.js

```diff
@@ -1,38 +1,40 @@
 import request from 'supertest';
 import { assert } from 'chai';
+import 'dotenv/config';
 
-import dotenv from 'dotenv';
+describe('face-register', () => {
+  const {
+    INSTANCE_NAME, AWS_BUCKET_NAME, AWS_S3_USER_IMAGE_KEY, TEST_USER_EMAIL1, TEST_USER_EMAIL2,
+    TEST_USER_PASSWORD
```

Comment:
Not to fix now, but for future reference: you can reassign the var during destructuring, like so:
const { firstUserEmail: TEST_USER_EMAIL } = process.env;